build.gradle

```diff
@@ -14,17 +14,18 @@ buildscript {
 
 plugins {
   id "com.jfrog.bintray" version "1.2"
-  id "us.kirchmeier.capsule" version "0.9.0"
+  //id "us.kirchmeier.capsule" version "1.0.2"
+//  id 'com.github.johnrengelman.shadow' version '7.1.2'
+  id 'java'
 }
 
 apply plugin: 'java'
 apply plugin: 'eclipse'
-apply plugin: 'maven'
 apply plugin: 'maven-publish'
 apply plugin: 'c'
 apply plugin: 'cpp'
 apply plugin: 'idea'
-apply plugin: 'javafx-gradle-plugin'
+//apply plugin: 'javafx-gradle-plugin'
```

Comment:
This can also go away, it was just used for packaging in ye olden days.

Reply:
Actually, only for packaging for Java Web Start, so not relevant anymore.

Reply:
Gone now.